packages/tenant-process/src/model/domain/models.ts

```diff
@@ -29,7 +29,8 @@ export type ApiCertifiedAttribute = z.infer<
   typeof api.schemas.CertifiedAttribute
 >;
 
-export const CertifiedAttributeQueryResult = api.schemas.CertifiedAttribute;
+export const CertifiedAttributeQueryResult =
+  api.schemas.CertifiedAttribute.strip();
```

Comment:
`api.schemas.CertifiedAttribute` is generated with the `passthrough` method. Meaning that zod does not strip the keys not "tracked" by the model.
This is a problem since we rely on [this parse](https://github.com/pagopa/interop-be-monorepo/blob/e14db64d17c71199d54c78841b7c00784ea23544/packages/tenant-process/src/services/readModelService.ts#L424) to remove the `lowername` we generate in the mongo query to order the results.

example
```typescript
// WITH PASSTHROUGH
const model = z.object({ name: z.string() }).passthrough()
const data = model.parse({ name: "Carmine", eta: 28 })

// eta keeps being into the object, even though the model does not have it
console.log(data) // { name: "Carmine", eta: 28 }

// WITHOUT PASSTHROUGH (default)
const model = z.object({ name: z.string() })
const data = model.parse({ name: "Carmine", eta: 28 })

// eta had been striped away
console.log(data) // { name: "Carmine" }
```

Reply:
when possible we should use internal types for query result parsing, to decouple persistence and api layers.

But nevertheless this is a good point. @MalpenZibo what do you think about adding `strip` to the handlebars template, as an additional precaution on sending out unexpected fields?